integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/OpenLineageRunEventBuilder.java

```diff
@@ -503,15 +516,35 @@ private OpenLineage.JobFacets buildJobFacets(
       List<Object> events,
       Collection<CustomFacetBuilder<?, ? extends JobFacet>> builders,
       OpenLineage.JobFacetsBuilder jobFacetsBuilder) {
-    events.forEach(event -> builders.forEach(fn -> fn.accept(event, jobFacetsBuilder::put)));
+    events.forEach(
+        event ->
+            builders.forEach(
+                fn ->
+                    openLineageContext
+                        .getMeterRegistry()
+                        .timer(
+                            "openlineage.spark.facets.job.execution.time",
```

Comment:
would it make sense to have a single class to store all the metric names as constants?

Reply:
No, but I think particular classes could hold that - however most often the name is used one time so I don't see very much use of it.